dotnet/src/SemanticKernel.Abstractions/SkillDefinition/NullReadOnlySkillCollection.cs

```diff
@@ -4,44 +4,54 @@
 
 namespace Microsoft.SemanticKernel.SkillDefinition;
 
-internal class NullReadOnlySkillCollection : IReadOnlySkillCollection
+internal sealed class NullReadOnlySkillCollection : IReadOnlySkillCollection
```

Comment:
I'm surprised this wasn't previously being flagged by https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1852

Reply:
I think we need to enable that manually. I added it to .editorconfig